app/views/relay/roundForm.scala

```diff
@@ -97,31 +97,23 @@ object roundForm:
         )(form3.flatpickr(_, minDate = None)),
         form3.checkbox(
           form("finished"),
-          raw("Completed"),
-          help = raw(
-            "Lichess detects round completion based on the source games. Use this toggle if there is no source."
-          ).some,
+          completed(),
+          help = completedHelp().some,
           half = true
         )
       ),
       form3.split(
         form3.group(
           form("delay"),
-          raw("Delay in seconds"),
-          help = frag(
-            "Optional, how long to delay moves coming from the source.",
-            br,
-            "Add this delay to the start date of the event. E.g. if a tournament starts at 20:00 with a delay of 15 minutes, set the start date to 20:15."
-          ).some,
+          delayInSeconds(),
+          help = delayInSecondsHelp().some,
           half = true
         )(form3.input(_, typ = "number")),
         isGranted(_.Relay) option
```

Comment:
this means it's only for team members, therefore there's no reason to translate it

Reply:
https://lichess.org/broadcast/round/{id}/edit

Reply:
I think this also needs translation. If you don't want it, I'll do what you want